src/server/resolve.rs

```diff
@@ -168,7 +187,12 @@ mod tests {
             unimplemented!();
         }
         fn get_store(&self, _: u64) -> Result<metapb::Store> {
-            Ok(self.store.clone())
+            // The store address will changed every second.
```

Comment:
s/changed/change on/

If the address change every 10ms, it will make the test case below run faster.